docker/.gitattributes

```diff
@@ -0,0 +1,3 @@
+Dockerfile.* text=auto
```

Comment:
Should we also do this with bash scripts and source files? I think `*.py` (for the Android build system), `*.rs` (for our Rust source files), and `*.sh` for our bash scripts would be good for consistency.

Reply:
do you mean the `text=auto`? [this](https://www.git-scm.com/docs/gitattributes#Documentation/gitattributes.txt-Settostringvalueauto) is what it does

> When `text` is set to "auto", the path is marked for automatic end-of-line conversion. If Git decides that the content is text, its line endings are converted to LF on checkin. When the file has been committed with CRLF, no conversion is done.